src/components/Login/Login.js

```diff
@@ -1,23 +1,30 @@
 import React from 'react';
-
+import './Login.css';
 function Login() {
   return (
     <div>
       <main className='main-content'>
-        <p>Login page</p>
       </main>
       <form method="post">
-        <label>
-          Email :
-          <input type="email" name="email" placeholder="email"/>
-        </label>
-        <br></br>
-        <label>
-          Password :
-          <input type="password" name="password" placeholder="password" />
-        </label>
+        <div className = "inputCenter">
+          <label>
+            <div className = "emailText">
+              Email :
+            </div>
+            <input type="email" name="email" placeholder="Email address"/>
+          </label>
+          <br></br>
+          <label>
+            <div className = "passwordText">
+              Password :
+            </div>
+            <input type="password" name="password" placeholder="••••••••••" />
```

Comment:
•••••••••• placeholder is not very descriptive. User's input should not be clear text but placeholder should advice user to type correct input